core/src/main/java/com/linecorp/armeria/internal/common/stream/AggregatingStreamMessage.java

```diff
@@ -79,6 +79,18 @@ public void close(Throwable cause) {
         abort(cause);
     }
 
+    @Override
+    public void abort() {
+        closed = true;
```

Comment:
I believe that this was a bug. Let's see the following situation:
```java
AggregatingDecodedHttpRequest req = ...
req.abort();
req.tryWrite(o);
```
Even though, the request is aborted, we can still write to the request because the `closed` isn't set to false.